docs/extensibility/extending-blocks.md

```diff
@@ -163,6 +163,20 @@ var withDataAlign = wp.compose.createHigherOrderComponent( function( BlockListBl
 wp.hooks.addFilter( 'editor.BlockListBlock', 'my-plugin/with-data-align', withDataAlign );
 ```
 
+### `editor.PostFeaturedImage.imageSize`
```

Comment:
This documentation should be located in a different place. It has no connection with extending blocks in the current shape.

Reply:
My bad, can you provide an alternative suggestion?

Reply:
#8358